FLEDGE.md

```diff
@@ -356,6 +356,8 @@ const myAuctionConfig = {
                          'https://example.fr': 'EUR',
                          '*': 'USD'},
   'sellerCurrency:' : 'CAD',
+  'deprecatedRenderURLReplacements':{{'${SELLER1}':'ssp1'},
+                                    {'%%SELLER2%%':'ssp2'}},
```

Comment:
To avoid confusion between SELLER1 corresponding to ssp1 while SELLER2 corresponding to ssp2:
```suggestion
  'deprecatedRenderURLReplacements':{{'${SELLER}':'exampleSSP'},
                                    {'%%SELLER_ALT%%':'exampleSSP'}},
```